src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -1488,6 +1488,34 @@ public async Task Locals()
         public async Task Parameters()
             => await VerifyItemExistsAsync(@"class c { void M(string args) { $$", "args");
 
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/55969")]
```

Comment:
📝 These tests are failing because the syntax tree for certain cases doesn't include any `GlobalStatementSyntax`. As a result, the semantic binder doesn't include any `InMethodBinder` instances, and the parameter is not available.

One example input which appears for testing is:

```
using System;

a
```

In this tree, `a` is part of an `IncompleteMemberSyntax` instead of a `GlobalStatementSyntax`. I'm not sure how, but it seems the compiler needs to allow for the case where the incomplete member is a top-level statement during symbol lookup.

Reply:
Wouldn't it be better to have the tests in place by changing the test input? eg would `Console.WriteLine($$` be work?

Reply:
Yes, but that's not the part that's broken. The tests are being implemented with the intent of saving the developer time who eventually fixes this (otherwise they would be required to implement the tests themselves). The original issue stays open after this merges.